triangles.js

```diff
@@ -0,0 +1,9 @@
+function isTriangle(side1, side2, side3) {
+  if ((side1 + side2) > side3 && (side2 + side3) > side1 && (side3 + side1) > side2 ) {
```

Comment:
When `if` statements return only boolean values, you can just return the `...` from `if(...)`